clog/clog_test.go

```diff
@@ -209,4 +214,39 @@ func TestLogger(t *testing.T) {
 		err := ToFileAtLevel("!/*&^%$", "debug")
 		So(err, ShouldNotBeNil)
 	})
+
+	Convey("Given a custom handler", t, func() {
+		ctx := context.Background()
+		handler, err := log15.SyslogHandler(syslog.LOG_USER,
+			"wrrunner", log15.LogfmtFormat())
+		So(err, ShouldBeNil)
+
+		Convey("it can log at given level", func() {
+			syslogpath := "/var/log/syslog"
+
+			if runtime.GOOS == "darwin" {
+				syslogpath = "/var/log/system.log"
+			} else if runtime.GOOS == "windows" {
+				t.Skip("skipping test; windows os not supported.")
+			}
+
+			tl, err := follower.New(syslogpath, follower.Config{
```

Comment:
Why is this variable called tl? Something more explanatory would be better.

Reply:
Removed, not needed